### ISSUE TITLE
Rewrited VARS in SCSS

### ISSUE THREAD

**Issue by DenisSokoloff** [twolfson/gulp.spritesmith#102]
I see rewrited VARS in SCSS

```
gulp.task("Create Atlas 1x & 2x", function ()
{
    gulp.src([filesAtlases1XPath + pngFilter, filesAtlases2XPath + pngFilter])
        .pipe(spritesmith(
            {
                imgName:        "atlas.png",
                imgPath:         "images/atlas.png",
                retinaImgName:  "atlas@2x.png",
                retinaImgPath:  "images/atlas@2x.png",
                retinaSrcFilter: [filesAtlases2XPath + pngFilter],
                cssName:        "scss/_images.scss"
            }))
        .pipe(gulp.dest(""))
        .on("end", function ()
        {
            gulp.start('Optimize Atlases');
        });
});
```

result: _images.scss with rewrited VARS (with same names)

`
....
$btn-home-name: 'btn-home';
$btn-home-x: 0px;
$btn-home-y: 80px;
$btn-home-offset-x: 0px;
$btn-home-offset-y: -80px;
$btn-home-width: 44px;
$btn-home-height: 44px;
$btn-home-total-width: 180px;
$btn-home-total-height: 124px;
$btn-home-image: 'images/atlas.png';
$btn-home: (0px, 80px, 0px, -80px, 44px, 44px, 180px, 124px, 'images/atlas.png', 'btn-home', );

...
$btn-home-name: 'btn-home';
$btn-home-x: 0px;
$btn-home-y: 160px;
$btn-home-offset-x: 0px;
$btn-home-offset-y: -160px;
$btn-home-width: 88px;
$btn-home-height: 88px;
$btn-home-total-width: 360px;
$btn-home-total-height: 248px;
$btn-home-image: 'images/atlas@2x.png';
$btn-home: (0px, 160px, 0px, -160px, 88px, 88px, 360px, 248px, 'images/atlas@2x.png', 'btn-home', );
...
`

**Comment by twolfson** [comment]
I don't understand this issue. Can you rephrase your question?

**Comment by twolfson** [comment]
Closing issue due to inactivtiy
